soccer/src/soccer/radio/packet_convert.cpp

```diff
@@ -1,15 +1,40 @@
 #include "packet_convert.hpp"
 
-#include <rj_geometry/util.hpp>
 #include <rj_common/status.hpp>
 #include <rj_common/time.hpp>
+#include <rj_geometry/util.hpp>
 
+#include "control/motion_setpoint.hpp"
 #include "robot_intent.hpp"
 #include "robot_status.hpp"
-#include "motion/motion_setpoint.hpp"
 
-// TODO(#1583): Make this a real ROS parameter
-constexpr float kMaxKickSpeed = 7.0;
+// TODO(#1583): Make these ROS parameters and move them to a central location
+constexpr double kMaxKickSpeed = 7.0;
+constexpr double kMinKickSpeed = 2.1;
+constexpr double kMaxChipSpeed = 3.0;
+constexpr double kMinChipSpeed = 0.5;
```

Comment:
Min max speed is different than that in `manipulator_control.cpp` so grsim will always be off until this todo is fixed.